### PR TITLE
Documentation refresh

### PR DESCRIPTION
**Pull request by ragusa87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,58 @@
...
+This project will help you to:
+
+Access docker containers via Traefik ([https://traefik.docker.lo](https://traefik.docker.lo)): reverse proxy and load balancer for containers
+
+Manage docker via Portainer ([https://portainer.docker.lo](https://portainer.docker.lo))
+
+## Prerequisite
+
+- Docker-compose
+- Dnsseq configured to forward \*.test domains on localhost.
```
Comment (Aerzas) [inline README.md]: There may be a typo to "dnsseq".

File: README.md [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
 Create certificates for HTTPS
+
 ```bash
 chmod u+x ./scripts/generate-certificates.sh
 ./scripts/generate-certificates.sh
 ```
-You need to add the generated certificate `certificates/docker.rootCA.crt` to your browser authorities and trust related websites.
 
-Start Traefik and Portainer
+You can add the generated certificate `certificates/docker.rootCA.crt` to your browser authorities and trust related websites.
```
Comment (Aerzas) [inline README.md]: We generate certificates named `pontsun.rootCA.crt`, don't we ?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,58 @@
...
+
+## Start Traefik and Portainer
+
 ```bash
 cd containers
+. .env
```
Comment (Aerzas) [inline README.md]: I'm not sure this is a required step.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,58 @@
...
 docker-compose up -d
 ```
 
-## Containers
+## Access the containers:
 
-- Traefik ([https://traefik.docker.lo](https://traefik.docker.lo)): reverse proxy and load balancer for containers
-- Portainer ([https://portainer.docker.lo](https://portainer.docker.lo)): GUI for Docker management
+- https://traefik.docker.test/
+- https://portainer.docker.test/#/home
```
Comment (Aerzas) [inline README.md]: Domain names are `*.ponstun.test`, aren't they ?

File: docs/installation-for-mac.md [hunk 5]
```diff
@@ -30,7 +29,7 @@ gem install docker-sync
 
 ## Dnsmasq
 
-Dnsmasq will automatically forward any **\*.docker.lo** domain to our
+Dnsmasq will automatically forward any **\*.docker.test** domain to our
```
Comment (Aerzas) [inline docs/installation-for-mac.md]: Same here, domain should be `pontsun.test`

File: docs/installation-for-mac.md [hunk 6]
```diff
@@ -39,7 +38,7 @@ brew install dnsmasq
 
 ```
 mkdir -pv $(brew --prefix)/etc/
-echo 'address=/docker.lo/127.0.0.1' > $(brew --prefix)/etc/dnsmasq.conf
+echo 'address=/docker.test/127.0.0.1' > $(brew --prefix)/etc/dnsmasq.conf
```
Comment (Aerzas) [inline docs/installation-for-mac.md]: Domain again

**Review by Aerzas — CHANGES_REQUESTED** [review]
Good idea to clean it up, there some obsoletes infos (not all coming from you proposal) which could be updated along the way probably.

File: README.md (hunk 2 above)
Comment (OdyX) [reply README.md]: Nope. Not in `master`

File: README.md (hunk 4 above)
Comment (OdyX) [reply README.md]: Not in `master` as of now.

**Comment by OdyX** [comment]
@Aerzas Rebased and split in 3 commits. Not everything is perfect, but the most obvious things are fixed.

**Pull request merged**